packages/pwa-kit-dev/package.json

```diff
@@ -55,6 +55,7 @@
     "@loadable/babel-plugin": "^5.13.2",
     "@loadable/server": "^5.15.0",
     "@loadable/webpack-plugin": "^5.15.0",
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.6",
```

Comment:
@vmarta  let's file this 3pp early, because I think it is possible that security rejects this 3pp.

Reply:
Submitted the 3PPs.